geopandas/geoseries.py

```diff
@@ -173,8 +222,15 @@ def copy(self, order='C'):
         copy : GeoSeries
         """
         # FIXME: this will likely be unnecessary in pandas >= 0.13
-        return GeoSeries(self.values.copy(order), index=self.index,
-                      name=self.name).__finalize__(self)
+        return GeoSeries(self._data,
+                         index=self.index, crs=self.crs,
+                         name=self.name)
```

Comment:
Currently this creates a copy of the block, but that's all.  It uses the same `VectorizedGeometry` data and pointers.  Inplace operations on a VectorizedGeometry object are going to be tricky.  This means that we will have to track references on the individual pointer level rather than on the VectorizedGeometry level.  Is it feasible to disallow inplace edits of a VectorizedGeometry object for a while?  This would be a deviation from Pandas expectations.